publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/config/RabbitMqPropertiesConfig.java

```diff
@@ -166,6 +177,23 @@ else if (key.contains("rabbitmq.createExchangeIfNotExisting")) {
 
         final String generateServerPath = (String) map.get(GENERATE_SERVER_PATH);
         generateURLTemplate.setGenerateServerPath(generateServerPath);
+
```

Comment:
Since we have this in App class: @PropertySource(value = "file:${catalina.home}/conf/config.properties", ignoreResourceNotFound = true)
You should be able to access these properties directly from the the environment object:
env.getProperty("some.property")

This should work also if applicationas.properties and config.yml is according to Spring documentations. Then it should work in all situations.